sources.json

```diff
@@ -647,5 +647,36 @@
     "type": "tooling",
     "tags": ["excel","i18n"],
     "license": "gpl-3.0"
+  },
+  {
+    "owner": "ui5-community",
+    "repo": "generator-ui5-library-webc",
+    "addedToBoUI5": "2023-08-25T00:00:00.007Z",
+    "type": "generator",
+    "tags": ["library","webcomponents","generator"],
+    "license": "apache-2.0"
+  }, {
+    "owner": "SAP",
+    "repo": "ui5-tooling-webc",
+    "addedToBoUI5": "2023-08-25T00:00:00.007Z",
+    "type": "module",
+    "tags": ["tooling","webcomponents","generator"],
+    "license": "apache-2.0"
+  },
+  {
+    "owner": "lemaiwo",
+    "repo": "ui5-space-webcomponent-library",
```

Comment:
i need in the root repo a file called `README.md` and `LICENSE`

Reply:
License added to the repo

Reply:
Sorry, do you mind renaming the readme to `README.md`
It seems this is the standard formatting for the file and my code is looking for this file format

Reply:
Done!